src/main/java/lotto/Util/LottoOption.java

```diff
@@ -0,0 +1,9 @@
+package lotto.Util;
+
+public class LottoOption {
+    public final static int LOTTO_TICKET_PRICE = 1000;
+    public static final String LOTTO_NUMBER_DELIMITER = ",";
+    public static final int LOTTO_MIN_NUMBER = 1;
+    public static final int LOTTO_MAX_NUMBER = 45;
+    public static final int LOTTO_MAX_DIGIT = 6;
+}
```

Comment:
```suggestion
public class LottoOption {
    public final static int LOTTO_TICKET_PRICE = 1000;
    public static final String LOTTO_NUMBER_DELIMITER = ",";
    public static final int LOTTO_MIN_NUMBER = 1;
    public static final int LOTTO_MAX_NUMBER = 45;
    public static final int LOTTO_MAX_DIGIT = 6;
    
    private LottoOption {
        // 불필요한 인스턴스화 방지
    }
}
```
상수로만 이어진 객체들은 외부에서 잘못해서 생성자를 호출하지 않게 인스턴스화 방지를 해주면 더 좋을 것 같아요!😁

Reply:
이런 생각은 못했던 것 같습니다
조언 감사합니다!